pgcli/packages/sqlcompletion.py

```diff
@@ -27,6 +28,15 @@ def suggest_type(full_text, text_before_cursor):
 
     identifier = None
 
+    def strip_named_query(txt):
+        pattern = re.compile(r'\\(n|nd|ns\s+[a-z]+)\s+')
```

Comment:
I don't think the regex is right. 

I think it should be 

```
        pattern = re.compile(r'^\s*\\ns\s+')
```

A few reasons:

1. Always match at the beginning. 
2. Anticipate a leading space in the front of the query. So if someone types `<space> \ns `, it should still work. 
3. \ns should be the only one that allows completion of sql query. \n and \nd should suggest names of the named queries.